### ISSUE TITLE
[lex] `\indextext{separate translation|see{compilation, separate}}` ?

### ISSUE THREAD

**Issue by Eisenwave** [cplusplus/draft#6843]
https://github.com/cplusplus/draft/blob/ceff4ea83b511be01a8e1756386ce6a2e06e323c/source/lex.tex#L7-L9

Is this indexed as intended? I find it strange that all these entries would redirect to `compilation, separate`.

Shouldn't it be the other way around? Do we even want to mention compilation here? After all, compilation isn't really the conventional wording we use here.

**Comment by jensmaurer** [comment]
I'd be ok with having all of these refer to "translation, phases"

Having commonly used terms (such as "separate compilation") in the index seems worthwhile even if it's not a normative term.

**Comment by frederick-vs-ja** [comment]
Also related to #5641. The index seemingly implies that the standard treats compilation same as translation, but it's unclear to me whether this is intended.

**Comment by jensmaurer** [comment]
Yes, the standard calls "translation" what the rest of the world might call "compilation", so we should mention "compilation" in the index.
